styles.css

```diff
@@ -291,11 +306,59 @@ h4 {
   nav {
     display: none;
   }
+
+	header {
+		justify-content: space-evenly;
```

Comment:
I would consider using `justify-content: space-between" so that the logo doesn't jump around at breakpoints.